<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>afterGameWinDialog_3</class>
 <widget class="QDialog" name="afterGameWinDialog_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>赛后新闻发布会</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>记者</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>在胜利过后，你会给球队什么奖励？</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>561</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>您</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>我答应过他们，会给他们一个小假期。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>这个问题我还在与我的助理讨论中。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>我会询问球员的意见。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
